tests/lang_types/var/tvar_arguments.nim

```diff
@@ -0,0 +1,200 @@
+discard """
+  target: "c js vm"
```

Comment:
IIRC the lang category runs on all targets by default.

Reply:
Yep, the `lang` category does, but the `lang_xxx` ones do not.